srcalc/src/main/webapp/WEB-INF/views/enterVariables.jsp

```diff
@@ -81,7 +81,7 @@
     <li>
     <%-- Add the required patientDfn parameter, preserving the patient from the current calculation. --%>
     <c:url var="newCalcUrl" value="/newCalc"><c:param name="patientDfn" value="${calculation.patient.dfn}"/></c:url>
-    <a class="btn-link" href="${newCalcUrl}">Start New Calculation</a></li>
+    <a href="${newCalcUrl}"><button type="button">Start New Calculation</button></a></li>
```

Comment:
Good change for consistency, but please emphasize the Run Calculation button now.